lib/src/main/java/com/auth0/android/lock/utils/HiddenField.java

```diff
@@ -0,0 +1,51 @@
+package com.auth0.android.lock.utils;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+public class HiddenField extends SignUpField {
+
+    private final String value;
+
+    /**
+     * Constructor for HiddenField instance
+     *
+     * @param key     the key to store this field as.
+     * @param value   the fixed value to set for this field
+     * @param storage the location to store this value into.
```

Comment:
Document the supported "keys" when the storage is "root profile". (or link to docs)

Reply:
Resolving because this is already added here https://github.com/auth0/Lock.Android/blob/master/lib/src/main/java/com/auth0/android/lock/utils/CustomField.java#L69